### DISCUSSION TITLE
Pre-built wheels

### DISCUSSION THREAD

**Discussion by darleybarreto** [Pylir/Pylir#25] [Ideas]
Hi there,

Is it possible to provide a pre-built wheels for Linux? They could be built and hosted here on GitHub on every release or some sort of nightly dev repo on conda having the last successful GH CI binaries.

Thanks!

**Comment by zero9178** [comment]
That would definitely be a good idea and good motivation to figure out a good release workflow. Nightly builds would probably also be very important until we have anything resembling production ready
